package.json

```diff
@@ -28,8 +28,8 @@
         "test": "./node_modules/.bin/mocha "
     },
     "dependencies": {
-        "axios": "0.21.1",
-        "bcryptjs": "^2.4.3",
+        "axios": "^0.21.1",
```

Comment:
How does this fix the issue ? Why not ensure it is the correct level to fix the issue raised ?